alf/algorithms/rl_algorithm.py

```diff
@@ -58,9 +55,9 @@ class RLAlgorithm(Algorithm):
     5. ``update_with_gradient()``: Do one gradient update based on the loss. It is
        used by the default ``train_iter()`` implementation. You can override to
        implement your own ``update_with_gradient()``.
-    6. ``calc_loss()``: calculate loss based the training_info collected from
-       ``rollout_step()`` or ``train_step()``. It is used by the default
-       implementation of ``train_iter()``. If you want to use the default
+    6. ``calc_loss()``: calculate loss based the ``experience`` and the ``train_info``
+       collected from ``rollout_step()`` or ``train_step()``. It is used by the
+       default implementation of ``train_iter()``. If you want to use the default
```

Comment:
perhaps we should also have a similar paragraph of description about Algorithm?

Reply:
added